tests/run_tests.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# Test runner to test the following examples:
+# bounds.c, set_bounds.c, general_bounds.c, xor_pointers.c.
+#
+# *** NOTE ***
+# The following examples do not need to be
+# tested because they are simply exploring features or
+# they are tests themselves/experiments:
+# allocate.c, check_length.c, check_mask.c, function.c,
+# seal.c, sentry.c, setjmp.c, stackscan.c
+set -e
+
+: "${BUILD_DIR:="../bin"}"
```

Comment:
Would we ever want this to be something other than `../bin/`?

Reply:
> In general (and we can experiment with this at squashing time) we prefer to have one commit that does the renaming and then subsequent commits which change things, to try and deal with the fact that git's renaming mechanism doesn't really exist (it works on content similarity, so if you rename + change, the rename is no longer considered a change).

Are you referring to "test_bounds.sh"?

Reply:
I think this comment ended up in the wrong place -- but yes.

Reply:
> I think this comment ended up in the wrong place -- but yes.

Yes, the place is wrong. So, what's your suggestion?

Reply:
> Would we ever want this to be something other than `../bin/`?

I've added the test script to .buildbot.

Reply:
> Yes, the place is wrong. So, what's your suggestion?

I meant: you replied to a PR-wide comment and put it back on a single line :) It should go in the general "chat" at the bottom of the PR.

Reply:
> >  Would we ever want this to be something other than ../bin/?
>I've added the test script to .buildbot.

I don't see how the two things are related?

Reply:
Oh, because buildbot wants it to be `bin`. OK, makes sense.